SConstruct

```diff
@@ -209,16 +210,25 @@ for path in module_search_paths:
 
 # Add module options.
 for name, path in modules_detected.items():
-    enabled = True
-    sys.path.insert(0, path)
-    import config
-
     try:
-        enabled = config.is_enabled()
-    except AttributeError:
-        pass
-    sys.path.remove(path)
-    sys.modules.pop("config")
+        if env_base["module_" + name + "_enabled"]:
+            enabled = True
+    except:
+        if not env_base["disable_all_modules"]:
+            enabled = True
```

Comment:
Without all those changes, this seems to work too:

```suggestion
            # Enable or disable modules by default.
            enabled = not env_base["disable_all_modules"]
```

Reply:
I originally thought that to be case, but (pointing out in your commit) [`mono`'s config.py will always override this setting with the code you have now, even if `module_mono_enabled=true` is set](https://github.com/Xrayez/godot/blob/modules-enabled-by-default/SConstruct#L218) which leads to [`env_base` being updated wrongly](https://github.com/Xrayez/godot/blob/modules-enabled-by-default/SConstruct#L228) (notice that the configurations are being _sent_ to `env_base`). On the same train of thought, I cannot do `module_bullet_enabled=true` for essentially the same reasons.

This led me to realize that I need to get the user input _before_ that happens and skip over it.

Reply:
Yeah that's an idea behind my commit, I'm not sure if a user would expect this overriding behavior. For your PR this logic makes sense I think.